src/main/java/core/basesyntax/Picture.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Picture {
```

Comment:
According to the checklist, class and interface names should not be verbs. In this case, 'Picture' suggests an object rather than an action. Consider renaming it to something that represents what this interface is about. For example, if it's meant to provide a drawing capability, you might name it 'Drawable' to better convey the purpose.